Sources/TuistDependencies/SwiftPackageManager/Utils/SwiftPackageManagerGraphGenerator.swift

```diff
@@ -321,14 +371,33 @@ extension PackageInfo.Target {
             }
         }
 
-        // TODO: map to ProjectDescription.Settings
-        return nil
+        var settingsDictionary: ProjectDescription.SettingsDictionary = [:]
+        if !headerSearchPaths.isEmpty {
```

Comment:
This is worth unit-testing to ensure we are setting the right build settings and prevent future regressions.

Reply:
Definitely! I have tried to open feature PRs soon to validate the approach, and now I am starting from the first one to add the required tests 🙌